### PR TITLE
Fix part of #2833 : Add missing named argument in StoryActivityTest

### PR DESCRIPTION
**Pull request by jkugsiya**
## Explanation

"Fixes part of #2833 : Add missing named argument in StoryActivityTest"
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt [hunk 1]
```diff
@@ -179,16 +179,19 @@ class StoryActivityTest {
   class TestApplication : Application(), ActivityComponentFactory, ApplicationInjectorProvider {
     private val component: TestApplicationComponent by lazy {
       DaggerStoryActivityTest_TestApplicationComponent.builder()
-        .setApplication(this)
+        .setApplication(context = this)
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: this is incorrect. 
PTAL - https://kotlinlang.org/docs/functions.html#named-arguments

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt [hunk 2]
```diff
@@ -179,16 +179,19 @@ class StoryActivityTest {
...
     }
 
     fun inject(storyActivityTest: StoryActivityTest) {
-      component.inject(storyActivityTest)
+      component.inject(storyActivityTest = storyActivityTest)
     }
 
     override fun createActivityComponent(activity: AppCompatActivity): ActivityComponent {
-      return component.getActivityComponentBuilderProvider().get().setActivity(activity).build()
+      return component.getActivityComponentBuilderProvider().get().setActivity(
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: I guess, we can keep this line unchanged.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
@jkugsiya Look at line number 116, these are the places where we need named arguments most, as passing simply 1 or any number doesn't help us understand what is the requirement of that argument. 

Reference - 
https://kotlinlang.org/docs/functions.html#named-arguments

**Comment by anandwana001** [comment]
@jkugsiya Please use the branch name a bit descriptive of what kind of work/issue this branch is solving. 
https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change
As for this PR, let it be, we cannot change the branch name now. But, please follow the same from the next PR's.

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt (hunk 1 above)
Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: > @jkugsiya Look at line number 116, these are the places where we need named arguments most, as passing simply 1 or any number doesn't help us understand what is the requirement of that argument.

As you said in line number 116 we need named arguments the most, but Android Studio is not allowing to make it a named argument by throwing an error that named arguments are not allowed in non-kotlin functions.
Screenshot 2021-03-16 135535

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: ah, what I missed here is `actionOnItemAtPosition` is a java function. Thanks for the update.

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt [hunk 3]
```diff
@@ -184,7 +184,7 @@ class StoryActivityTest {
     }
 
     fun inject(storyActivityTest: StoryActivityTest) {
-      component.inject(storyActivityTest)
+      component.inject(storyActivityTest = storyActivityTest)
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: we can skip this as this is already readable.

**Review by anandwana001 — CHANGES_REQUESTED** [review]

**Comment by anandwana001** [comment]
Re running checks

**Comment by jkugsiya** [comment]
> Re running checks

Can you please help me with the one check that is failing constantly?

**Review by anandwana001 — APPROVED** [review]
LGTM

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt [hunk 4]
```diff
@@ -132,7 +132,7 @@ class StoryActivityTest {
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(storyActivityTest = this)
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: We dont need named argument for this line.

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt [hunk 5]
```diff
@@ -73,7 +73,7 @@ import javax.inject.Singleton
 
 /** Tests for [StoryActivity]. */
 @RunWith(AndroidJUnit4::class)
-@LooperMode(LooperMode.Mode.PAUSED)
+@LooperMode(value = LooperMode.Mode.PAUSED)
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: We dont need named argument for this line.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@jkugsiya Suggested change.

**Comment by anandwana001** [comment]
@jkugsiya Please don't resolve the conversation started by reviewers. The conversation should only be resolved by the one who has started it.

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt (hunk 4 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: @jkugsiya Please reply to this comment.

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt (hunk 5 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: @jkugsiya Please reply to this comment.

Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: https://github.com/oppia/oppia-android/pull/2903/commits/8ba9f9b8e206dc434c26d0e03a846bc8081e1a09
This requested change is satisfied in the above linked commit.

File: app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt (hunk 4 above)
Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/story/StoryActivityTest.kt]: https://github.com/oppia/oppia-android/commit/8ba9f9b8e206dc434c26d0e03a846bc8081e1a09
This requested change is provided in the above linked commit!

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**